src/index.js

```diff
@@ -1,47 +1,12 @@
 import './style.css';
 import './fontawesome/css/all.css';
-
-const tasks = [
```

Comment:
The task object's values for `completed` should be updated to true in local storage when the user checks it off from the list.